src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator1 = {
+    result: 0,
+    add(num) {
+      calculator1.result += num;
+    },
+    subtract(num) {
+      calculator1.result -= num; ;
+    },
+    multiply(num) {
+      calculator1.result *= num; ;
+    },
+    divide(num) {
+      calculator1.result /= num;
+    },
+    reset() {
+      calculator1.result = 0;
+
+      return calculator1;
+    },
+    operate(operand, number) {
+      operand(number);
```

Comment:
```suggestion
      operand.call(this, number);
```
here can be problem with this in operand method, when you will use this in methods. so you can set this directly with call method.